eth/polls/polllogs.go

```diff
@@ -89,6 +97,10 @@ func (p *EthLogPoll) AllLogs(state loomchain.ReadOnlyState,
 		return nil, errors.New("Filter FromBlock is greater than ToBlock")
 	}
 
+	if end-start > p.maxBlockRange {
+		return nil, fmt.Errorf("max allowed block range (%d) exceeded", p.maxBlockRange)
+	}
```

Comment:
The issue here is that `EthLogPoll.AllLogs` is called from `QueryServer.EthGetFilterLogs` but when you call `EthGetFilterLogs` you can't specify the block range, you can only specify the block range when you call `QueryServer.EthNewFilter`... which makes all this rather frustrating to figure out for end users since the `latest` block is always increasing. So, instead of returning an error the end of the range should be auto-adjusted in `EthLogPoll.AllLogs`.

Reply:
Done